### DISCUSSION TITLE
update not working

### DISCUSSION THREAD

**Discussion by DerMeso** [netbox-community/netbox-docker#735] [Getting Help]
I try to update to the latest version, every guide I use.

https://github.com/netbox-community/netbox-docker/wiki/Troubleshooting
https://github.com/netbox-community/netbox-docker/wiki/Updating

does not work, it does not update the image. I have followed both instructions exactly.

**Comment by kkthxbye-code** [comment]
No one can help you when you provide basically no information. I told you the exact same thing here:

https://github.com/netbox-community/netbox/discussions/8989#discussioncomment-2459854

Please read: http://www.catb.org/~esr/faqs/smart-questions.html

**Reply by DerMeso** [reply]
I am trying to update my netbox from version 86bdec7313df (v3.1.10) to v3.2

I first update the REPO

```

git checkout release &&
git pull -p origin release

```
Server then reports

```
from https://github.com/netbox-community/netbox-docker
 * branch release -> FETCH_HEAD
```

```
`git pull origin release`

docker-compose rm -fs netbox netbox-worker
docker-compose pull
docker-compose up -d netbox netbox-worker
```


Unfortunately, it still shows me after that that the version is the
